### DISCUSSION TITLE
Nuxt is generating html (in stead of js) for _nuxt/entry.mjs

### DISCUSSION THREAD

**Discussion by HendrikJan** [nuxt/nuxt#16726] [Questions]
Hi,

I have a Nuxt3 setup where every file returned from `_nuxt` (files like `_nuxt/entry.mjs` and `_nuxt/@vite/client`) return the following contenct:
```html
<!DOCTYPE html>
<html >

<head >
  
</head>

<body >
  <div id="__nuxt"></div><script>window.__NUXT__={serverRendered:false,config:{public:{},app:{baseURL:"\u002F",buildAssetsDir:"\u002F_nuxt\u002F",cdnURL:""}}}</script><script type="module" src="/_nuxt/@vite/client"></script><script type="module" src="/_nuxt/entry.mjs"></script>
</body>

</html>
```

This must have something to do with being behind a reverse proxy.  
When running without proxy, Nuxt will return valid js.

I've added a logger to `/server/middleware:
```js
export default defineEventHandler((event) => {
    console.log('📡 ' + event.req.url)
})
```

The result in the terminal shows that the urls that Nuxt recieves are correct:
```
📡 /_nuxt/@vite/client
📡 /_nuxt/entry.mjs
```

Does anybody have an idea why things are not working behind a proxy?

**Comment by Flanker72** [comment]
Seems that your proxy rewrites any requests to your "index" path, causing every file rendered like this.
Please, check your proxy settings, it seems to be not a Nuxt-related issue.

**Reply by HendrikJan** [reply]
You might be right.
I simplified the proxy as much as possible an used a fresh new Nuxt3 installation, and Nuxt is now working without problems (so problem could still be proxy or something in my Nuxt setup).
I'll copy my original project into this until it breaks -- to find out what the cause of my problem was.

**Reply by HendrikJan** [reply]
The problem was not solved by changing the proxy and seems not to be caused by a wrongly configured proxy.

I have created an issue with a reproduction here: https://github.com/nuxt/framework/issues/5699

**Comment by HendrikJan** [comment]
An answer to an issue that fixed the problem (at least part of it) can be found here: https://github.com/vitejs/vite/issues/8941
